lib/omniauth/strategy.rb

```diff
@@ -208,9 +208,9 @@ def request_call # rubocop:disable CyclomaticComplexity, MethodLength, Perceived
         call_app!
       else
         if request.params['origin']
-          env['rack.session']['omniauth.origin'] = request.params['origin']
+          session['omniauth.origin'] = request.params['origin']
```

Comment:
It isn't nope. It seems to use the default attribute accessor so it basically does the same thing. I updated it for clarity and it does not break the test suite, that's good enough for me ^^